packages/swarm-components/src/Icon.test.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+expect.extend({ toMatchImageSnapshot });
+
+import getScreenRenderer from './testUtils/screenRenderer';
+
+import Icon, { MEDIA_SIZES } from './Icon';
+
+describe('Icon', () => {
+	let renderer;
+
+	// This is ran when the suite starts up.
+	beforeAll(async () => {
+		renderer = await getScreenRenderer({
+			viewport: { width: 400, height: 500 },
+			// verbose: true,
+		});
+	});
+
+	// This is ran when the suite is done - stop the renderer.
+	afterAll(() => {
+		// comment next line out if you want to open it in your browser for debugging
+		return renderer.stop();
+	});
+
+	it('Should match screenshot', async () =>
+		expect(
+			await renderer.screenshot(
+				<div>
+					<h2>Regular</h2>
```

Comment:
If you update with master, there is a `getTestMarkup` helper function in `testUtils.jsx`. Might be easier/DRY-er to use it but not strictly necessary